<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroClass</class>
 <widget class="QMainWindow" name="IntroClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>383</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>382</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3D Scanner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2,0,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="mainlabel">
        <property name="minimumSize">
         <size>
          <width>415</width>
          <height>280</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>415</width>
          <height>280</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="mainwin.qrc">:/Resources/Resources/3D scanner.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_Viewer">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Viewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Startscan">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Start Scan</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>435</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="menuQuit"/>
   </widget>
   <widget class="QMenu" name="menutools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="menuViewer"/>
    <addaction name="menuScanner"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menutools"/>
  </widget>
  <action name="menuQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="menuViewer">
   <property name="text">
    <string>Viewer</string>
   </property>
  </action>
  <action name="menuScanner">
   <property name="text">
    <string>Scanner</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwin.qrc"/>
 </resources>
 <connections/>
</ui>
